### PR TITLE
[GUI] clean MenuManager.cpp

### PR DESCRIPTION
**Pull request by mosfet80**
Clean dead code

### CODE REVIEW

File: src/Gui/MenuManager.cpp [hunk 1]
```diff
@@ -205,46 +205,7 @@ void MenuManager::setup(MenuItem* menuItems) const
...
-    //
-    // This works as expected for action adding to a toolbar. As Qt will ignore
-    // actions inside an invisible toolbar.  However, Qt refuse to do the same
-    // for actions in a hidden menu action of a menu bar. This is very likely a
-    // Qt bug, as the behavior does not seem to conform to Qt's documentation
-    // of Qt::ShortcutContext.
-    //
-    // Clearing the menu bar, and recreate it every time when switching
-    // workbench with only the active actions can solve this problem.
-    menuBar->clear();
```
Comment (wwmayer) [inline src/Gui/MenuManager.cpp:246]: This is NOT dead code.

Comment (oursland) [reply src/Gui/MenuManager.cpp:246]: @wwmayer do you mean just the `menuBar->clear()`? If so, git chose to keep the previous instance of that in the diff.

Or is the `#if 0`'d code still valid?

Comment (wwmayer) [reply src/Gui/MenuManager.cpp:246]: > do you mean just the menuBar->clear()

Yes. Using a text editor with syntax highlighting you can easily see that this line is active code. And of course the comment block above must stay too

**Comment by chennes** [comment]
Closing for now, we want to keep that large comment.

**Pull request closed without merging**